app/Listeners/MetricsListener.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * SPDX-License-Identifier: MIT
+ * Copyright (c) 2017-2018 Tobias Reich
+ * Copyright (c) 2018-2025 LycheeOrg.
+ */
+
+namespace App\Listeners;
+
+use App\Events\Metrics\BaseMetricsEvent;
+use App\Models\Configs;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * Just logging of Cache events.
+ */
+class MetricsListener
+{
+	/**
+	 * Handle the event.
+	 */
+	public function handle(BaseMetricsEvent $event): void
+	{
+		if (Configs::getValueAsBool('metrics_enabled') === true) {
+			// Increment the respective metric in the database
+			DB::table('statistics')
```

Comment:
Direct call with `Statistics::increment()`? instead of `DB::` facade?

Reply:
That would also work, but on the DB has less redirections (rather than fetching the table etc).
No strong opinion.